settings.py

```diff
@@ -355,7 +353,14 @@
 TERRA_PERMS_CACHE_EXPIRE_SECONDS = os.environ.get('TERRA_PERMS_CACHE_EXPIRE_SECONDS', 60)
 TERRA_WORKSPACE_CACHE_EXPIRE_SECONDS = os.environ.get('TERRA_WORKSPACE_CACHE_EXPIRE_SECONDS', 300)
 
+SERVICE_ACCOUNT_FOR_ANVIL = None
+
 if TERRA_API_ROOT_URL:
+    SERVICE_ACCOUNT_FOR_ANVIL = subprocess.run(['gcloud auth list --filter=status:ACTIVE --format="value(account)"'],
+                                               capture_output=True, text=True, shell=True).stdout.split('\n')[0]
+    if not SERVICE_ACCOUNT_FOR_ANVIL:
+        raise Exception('Login Google service account with command "$ gcloud auth activate-service-account" before running seqr')
```

Comment:
We should NEVER manually run this command. If seqr is deployed without the google account being authenticated, it means something has gone horribly wrong with our docker image. Pease replace this message with something like. "Error starting seqr - gcloud auth is not properly configured"